client/extensions/woocommerce/state/sites/settings/stripe-connect-account/test/handlers.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	createAccount,
+} from '../actions.js';
+import {
+	handleAccountCreate,
+	handleAccountCreateSuccess,
+	handleAccountCreateFailure,
+} from '../handlers.js';
+import {
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_COMPLETE,
+} from 'woocommerce/state/action-types';
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+
+describe( 'handlers', () => {
+	describe( '#handleCreateAccountRequest', () => {
+		it( 'should dispatch a POST request', () => {
+			const siteId = '123';
+			const email = 'foo@bar.com';
+			const country = 'US';
+			const dispatch = spy();
+			const action = createAccount( siteId, email, country );
+			handleAccountCreate( { dispatch }, action );
+			expect( dispatch ).to.have.been.calledWithMatch( {
+				type: WPCOM_HTTP_REQUEST,
+				body: JSON.stringify( { email, country } ),
+				method: 'POST',
+				path: `/jetpack-blogs/${ siteId }/rest-api/`,
+				query: {
+					path: '/wc/v1/connect/stripe/account/&_method=POST',
+					json: true,
+					apiVersion: '1.1',
+				}
+			} );
+		} );
+	} );
+
+	describe( '#handleAccountCreateSuccess()', () => {
+		it( 'should dispatch create account receive on success with the account info', () => {
+			const siteId = '123';
+			const email = 'foo@bar.com';
+			const countryCode = 'US';
+			const store = {
+				dispatch: spy(),
+			};
+			const response = {
+				data: {
+					account_id: 'acct_14qyt6Alijdnw0EA',
+					success: true,
+				}
+			};
+
+			const action = createAccount( siteId, email, countryCode );
+			handleAccountCreateSuccess( store, action, response );
+
+			expect( store.dispatch ).calledWith( {
+				type: WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_COMPLETE,
+				connectedUserID: response.data.account_id,
+				email,
+				siteId,
+			} );
+		} );
+	} );
+
+	describe( '#handleAccountCreateFailure()', () => {
+		it( 'should dispatch create account error', () => {
+			const siteId = '123';
+			const email = 'foo@bar.com';
+			const countryCode = 'US';
+			const store = {
+				dispatch: spy(),
+			};
+			const response = {
```

Comment:
You could save a few lines here and just pass in a simple string into `handleAccountCreateFailure`, and make sure that gets set to `error`. You are really only using the `message` part of this response.